islandora_plupload.module

```diff
@@ -44,7 +44,7 @@ function islandora_plupload_form_alter(&$form, &$form_state, $form_id) {
 /**
  * Recursively change managed_file elements to plupload.
  *
- * @param array $form
+ * @param array|null $form
```

Comment:
Seems... spooky... shrug? Assuming it was complaining about the missing type hint, and [we were protecting against something there](https://github.com/discoverygarden/islandora_plupload/pull/16/files#diff-d1c2ac226ae0e52fadd210d666dc1883L52)... just yeah, spooky.